kpops/utils/pydantic.py

```diff
@@ -21,12 +22,79 @@ def to_dot(s: str) -> str:
     return s.replace("_", ".")
 
 
-class CamelCaseConfig(BaseConfig):
-    alias_generator = to_camel
-    allow_population_by_field_name = True
+def by_alias(field_name: str, model: BaseModel) -> str:
+    """Return field alias if exists else field name.
 
+    :param field_name: Name of the field to get alias of
+    :param model: Model that owns the field
+    """
+    return model.model_fields.get(field_name, Field()).alias or field_name
 
-class DescConfig(BaseConfig):
-    @classmethod
-    def schema_extra(cls, schema: dict[str, Any], model: type[BaseModel]) -> None:
+
+def exclude_by_value(
+    dumped_model: dict[str, Any], *excluded_values: Any
+) -> dict[str, Any]:
+    """Strip all key-value pairs with certain values.
```

Comment:
what's the use case?

Reply:
A more generic version of `exclude_none`. [Use case](https://github.com/bakdata/kpops/blob/refactor/pydantic-v2/kpops/components/streams_bootstrap/streams/model.py#L84)